volatility3/framework/plugins/windows/callbacks.py

```diff
@@ -249,7 +514,7 @@ def list_registry_callbacks(
                 "CmpCallBackCount",
                 "CallbackListHead",
             ]
-            vollog.debug("Failed to get registry callbacks!")
+            # vollog.debug("Failed to get registry callbacks!")
```

Comment:
Again, debugging information shouldn't affect normal usage, so I'd prefer it were there for developers to figure out when things go wrong.  If it's too verbose, that's fine and we can look for a better way to get the right information across, but if this was just a forgotten comment then it's be best to reinstate the message if possible (happy to change it to `constants.LOGLEVEL_VVVV` if that'd be better?